apps/site-landing/src/containers/MainNavigation/styles.module.less

```diff
@@ -1,9 +1,100 @@
-.mainMenu{
-    width: auto;
-    display: inline-block;
-    height: 60px;
-    margin-bottom: 50px;
-    .ui.secondary.pointing.menu{
-        border-bottom: none;
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+
+/** linear interpolate value from small to big as screen goes from small-size to big-size */
+.scalable-unit(@property, @small, @big, @unit) {
```

Comment:
In the very near future, scalable-unit will be moved into core semantic code, available for use throughout the project.